clients/src/main/java/org/apache/kafka/common/requests/CreateTopicsRequest.java

```diff
@@ -167,9 +176,12 @@ private CreateTopicsRequest(Map<String, TopicDetails> topics, Integer timeout, s
         }
         struct.set(REQUESTS_KEY_NAME, createTopicRequestStructs.toArray());
         struct.set(TIMEOUT_KEY_NAME, timeout);
+        if (version >= 1)
```

Comment:
If `validateOnly` is true, I wonder if we should raise an exception if the version is 0. It seems a little dangerous otherwise.